src/app.test.js

```diff
@@ -12,4 +12,29 @@ describe('#app', () => {
         expect(response.status).toBe(200);
         expect(response.body).toMatchObject({ status: 'ok' });
     });
+
+    test('it should startup check kafka health and return health check status ok', async () => {
+        // Mocking kafkajs library
+        jest.mock('kafkajs', () => {
+            //Mock the connect, disconnect method of the producer
+            return {
+                producer: jest.fn().mockImplementationOnce(() => {
+                    return {
+                        connect: jest.fn().mockImplementationOnce(() => { return Promise.resolve('connected');}),
+                        disconnect: jest.fn().mockImplementationOnce(() => { return Promise.resolve('disconnected');})
+                    };
+                })
+            };
+        });
+
+        // Setting ENABLE_EVENTS_KAFKA to '1' which implies kafka is being used.
+        process.env.ENABLE_EVENTS_KAFKA = '1';
+
+        let request = await createTestRequest();
+        let response = await request.get('/health');
+        expect(response.status).toBe(200);
+        // If Kafka is healthy status returned is ok
+        expect(response.body).toMatchObject({ status: 'ok' });
+        process.env.ENABLE_EVENTS_KAFKA = '0';
```

Comment:
Instead of setting the ENABLE_EVENTS_KAFKA to 0, we should set it to a value that it was set to previously. This requires storing its previous value before setting it to 1.